### PR TITLE
Rgp tests on

### PR DESCRIPTION
**Pull request by Rinchan20**
PR does a couple of things:

- Turns on RGP tests (I previously thought they were already turned on)
- Makes changes to the tests to make them a bit more robust (so that many family members can be added to a family account)
- Slight tweak to KitUpload.waitForReady() function to account for studies whose expected kit types are not Saliva & Blood e.g. RGP currently has the expected kit types of Blood and Blood&RNA

below is an example pic to show playwright created family members:
Screenshot 2023-09-21 at 10 55 03 AM

### CODE REVIEW

File: playwright-e2e/tests/dsm/kitUploadFlow/blood-and-rna-kit-upload-flow.spec.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ import { saveParticipantGuid } from 'utils/faker-utils';
 import { ParticipantListTable } from 'dsm/component/tables/participant-list-table';
 
 test.describe('Blood & RNA Kit Upload', () => {
-test.skip('Verify that a blood & rna kit can be uploaded @dsm @rgp @functional @upload', async ({ page, request}, testInfo) => {
+test('Verify that a blood & rna kit can be uploaded @dsm @rgp @functional @upload', async ({ page, request}, testInfo) => {
```
Comment (aweng98) [inline playwright-e2e/tests/dsm/kitUploadFlow/blood-and-rna-kit-upload-flow.spec.ts]: indent space look wrongs

File: playwright-e2e/tests/rgp/dsm-family-enrollment.spec.ts [hunk 2]
```diff
@@ -12,12 +12,13 @@ import { saveParticipantGuid } from 'utils/faker-utils';
...
 import { calculateAge } from 'utils/date-utils';
 import { logInfo } from 'utils/log-utils';
+import { v4 as uuid } from 'uuid';
 
 
 test.describe.serial('DSM Family Enrollment Handling', () => {
     let rgpEmail: string;
 
-    test.skip('Verify the display and functionality of family account dynamic fields @dss @functional @rgp', async ({ page, request}) => {
+    test('Verify the display and functionality of family account dynamic fields @dss @functional @rgp', async ({ page, request}) => {
```
Comment (aweng98) [inline playwright-e2e/tests/rgp/dsm-family-enrollment.spec.ts]: indent space looks wrong

File: playwright-e2e/tests/rgp/dsm-family-enrollment.spec.ts [hunk 3]
```diff
@@ -606,7 +607,7 @@ test.describe.serial('DSM Family Enrollment Handling', () => {
     expect(maternalGrandfatherFamilyID).toBe(probandFamilyID);
     });
 
-    test.skip('Verify that a family member can be added using copied proband info @dss @rgp @functional', async ({ page, request }) => {
+    test('Verify that a family member can be added using copied proband info @dss @rgp @functional', async ({ page, request }) => {
```
Comment (aweng98) [inline playwright-e2e/tests/rgp/dsm-family-enrollment.spec.ts]: tag is `@dsm`, right?

File: playwright-e2e/tests/rgp/dsm-family-enrollment.spec.ts [hunk 4]
```diff
@@ -85,7 +86,7 @@ test.describe.serial('DSM Family Enrollment Handling', () => {
     });
 
     //Skipping until housekeeping stuff is fixed
-    test.skip('Verify that the proband family member tab can be filled out @dss @functional @rgp @proband', async ({ page, request }) => {
+    test('Verify that the proband family member tab can be filled out @dss @functional @rgp @proband', async ({ page, request }) => {
```
Comment (aweng98) [inline playwright-e2e/tests/rgp/dsm-family-enrollment.spec.ts]: tag is `@dsm`, right?

File: playwright-e2e/tests/dsm/kitUploadFlow/blood-and-rna-kit-upload-flow.spec.ts [hunk 5]
```diff
@@ -24,7 +24,7 @@ import { saveParticipantGuid } from 'utils/faker-utils';
 import { ParticipantListTable } from 'dsm/component/tables/participant-list-table';
 
 test.describe('Blood & RNA Kit Upload', () => {
-test.skip('Verify that a blood & rna kit can be uploaded @dsm @rgp @functional @upload', async ({ page, request}, testInfo) => {
+  test('Verify that a blood & rna kit can be uploaded @dsm @rgp @functional @upload', async ({ page, request}, testInfo) => {
```
Comment (aweng98) [inline playwright-e2e/tests/dsm/kitUploadFlow/blood-and-rna-kit-upload-flow.spec.ts:27]: not working.

Screenshot 2023-09-20 at 7 05 06 PM

**Comment by aweng98** [comment]
Just realize this now, PR description is empty. Could you add some description? Maybe a video recording of playback.

File: playwright-e2e/tests/dsm/kitUploadFlow/blood-and-rna-kit-upload-flow.spec.ts (hunk 5 above)
Comment (Rinchan20) [reply playwright-e2e/tests/dsm/kitUploadFlow/blood-and-rna-kit-upload-flow.spec.ts:27]: forgot to put an await - should be fixed now

**Review by aweng98 — APPROVED** [review]

**Pull request merged**